[2025-03-21T17:44:35.608+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T17:44:35.648+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather-data-pipeline.check_and_insert_data manual__2025-03-21T17:44:21.784012+00:00 [queued]>
[2025-03-21T17:44:35.665+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather-data-pipeline.check_and_insert_data manual__2025-03-21T17:44:21.784012+00:00 [queued]>
[2025-03-21T17:44:35.666+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-21T17:44:35.687+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): check_and_insert_data> on 2025-03-21 17:44:21.784012+00:00
[2025-03-21T17:44:35.696+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=221) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-21T17:44:35.699+0000] {standard_task_runner.py:72} INFO - Started process 223 to run task
[2025-03-21T17:44:35.700+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather-data-pipeline', 'check_and_insert_data', 'manual__2025-03-21T17:44:21.784012+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/weather-data-pipeline.py', '--cfg-path', '/tmp/tmp53hk9c6z']
[2025-03-21T17:44:35.703+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask check_and_insert_data
[2025-03-21T17:44:36.063+0000] {task_command.py:467} INFO - Running <TaskInstance: weather-data-pipeline.check_and_insert_data manual__2025-03-21T17:44:21.784012+00:00 [running]> on host c25e9da6226b
[2025-03-21T17:44:36.259+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather-data-pipeline' AIRFLOW_CTX_TASK_ID='check_and_insert_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T17:44:21.784012+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T17:44:21.784012+00:00'
[2025-03-21T17:44:36.261+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T17:44:36.302+0000] {base.py:84} INFO - Retrieving connection 'rizqi-neon'
[2025-03-21T17:44:45.025+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T17:44:45.066+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T17:44:45.068+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather-data-pipeline, task_id=check_and_insert_data, run_id=manual__2025-03-21T17:44:21.784012+00:00, execution_date=20250321T174421, start_date=20250321T174435, end_date=20250321T174445
[2025-03-21T17:44:45.160+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T17:44:45.195+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-21T17:44:45.198+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
